docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,119 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component: changes stateprop1 from false to true which changes some text. 
+
+```
+import React, { useReducer } from 'react';
+import * as ACTIONS from '../store/actions'
```

Comment:
Can we simplify this down to just one file, with no special constants or anything? Just `dispatch({ type: 'success'} )`. It helps to have the bare minimum in these examples.